metadata/metadata.go

```diff
@@ -1476,7 +1476,8 @@ func (serv *MetadataServer) genericCreate(ctx context.Context, res Resource, ini
 	if err := resourceNamedSafely(id); err != nil {
 		return nil, err
 	}
-	if existing, err := serv.lookup.Lookup(id); err == nil {
+	existing, _ := serv.lookup.Lookup(id)
```

Comment:
Why cancel the error out?

Reply:
I waffled on that. I thought that the previous error was due to an existing resource, so now the presence/absence of the resource determines behavior, but errors could occur for other reasons, too. I'll add that back. Thanks for catching this.